### ISSUE TITLE
about Loss

### ISSUE THREAD

**Issue by zy937085318** [zchrissirhcz/pytorch_hed#1]
Hi, with ur log file, I find all losses are not decreased steadily while the Keras hed model does not have this situation, do u know what's the problem?

**Comment by zchrissirhcz** [comment]
Hi, @zy937085318 , which keras HED implementation do you use? And what batchSize do you use?

**Comment by zchrissirhcz** [comment]
Large batch size will makes the loss curve more smooth.
In [Keras_HED](https://github.com/lc82111/Keras_HED) the author use default batch size = 10, which will obviously makes the loss decrease steadily.

**Comment by zy937085318** [comment]
Thx so much for ur reply, keras model I used is [https://github.com/lc82111/Keras_HED](url).  The loss of keras hed would drop dramatically in the first 1000 steps when I set batch_size to 1. However, the pytorch hed's losses always lingered near 0.05 and seem not converged. Both keras and pytorch hed models are applied with Adam optimizer. Do u have any idea about the difference, it confuses me for several days and I plan to rewrite the loss function of pytorch hed model :(

**Comment by zchrissirhcz** [comment]
@zy937085318 Hi, how do you observe the training loss, of Keras_HED and Pytorch_HED? I tried to train Keras_HED and found it with a tensorboard, but plot loss in a very low frequency and I don't know where to change that.

I'll try to evaluate the trained Pytorch_HED, and train tensorpack's HED and evaluate it.

**Comment by zchrissirhcz** [comment]
@zy937085318 Actually, I've tried to evaluate Pytorch_HED and official RCF's trained result. They spend 10 and 7 hours respectively.
